R/get_negative_cases.R

```diff
@@ -0,0 +1,22 @@
+#' Find all observations with negative incidence in a
+#' given data frame
+#'
+#' @param data data frame
```

Comment:
Let's specify a little more detail about the data frame (what is in it?  what are the columns?)